packages/popup/src/pages/CurrentSeed/index.js

```diff
@@ -3,39 +3,52 @@ import Input from '@cfxjs/component-input'
 import Button from '@cfxjs/component-button'
 import {Selected} from '@cfxjs/component-icons'
 import {CompWithLabel} from '../../components'
-import create from '../../hooks/zustand.js'
+import create from '../../hooks/zustand'
 
 const useStore = create(
   (set, get) => ({
     // value
     hdGroup: [],
     creatingAccount: false,
-    accountCreatationError: null,
+    accountCreationError: null,
     accountName: '',
-    selectedGroup: null,
+    accountNamePlaceholder: '',
+    selectedGroupIdx: 0,
 
     // hook
     groupAfterSet: ({groupData}) => {
-      set({hdGroup: groupData.filter(({vault: {type}}) => type === 'hd')})
-      if (!get().selectedGroup) set({selectedGroup: get().group.groupData[0]})
+      const hdGroup = groupData.filter(g => g?.vault?.type === 'hd')
+      set({hdGroup})
+      if (hdGroup[0]) {
```

Comment:
is should be selectedGroupIdx here, the placeholder should be changed when user select another group